src/logic/dictionaryUtils.ts

```diff
@@ -104,7 +128,7 @@ export function matchesToPlayersHelper(
     commanderNameFilter?: string,
     startDate?: Date,
 ): { [id: string]: Player } {
-    const filteredMatches = filterMatchesByDate(matches, startDate);
+    const filteredMatches = filterMatchesBySeatCount(filterMatchesByDate(matches, startDate), 4);
```

Comment:
same comment here as above-- we don't want to exclude these matches from being visible, just from being included in calculations. 

Moving forward, we need to audit the site  to make sure we aren't doing any calculations on maches.length.

Reply:
(the audit is not something that you need to do, it would be part of the larger refactor)